client/src/components/InternshipTag.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import { Box, useTheme } from "@mui/material";
+
+const InternshipTag = (props) => {
+  let theme = useTheme();
+
+  return (
+    <>
+      <Box
+        display="flex"
+        maxWidth={false}
+        sx={{
+          borderRadius: "0.5rem",
+          padding: "0.5rem 1rem",
+          bgcolor: theme.palette.blue.lighter,
+          color: theme.palette.text.primary,
+          border: 1,
+          maxWidth: "fit-content",
+        }}
+      >
+        <props.icon />
```

Comment:
If we're passing an element into the props for this, we could destructure it from props first and then render it with the <> like a component

```jsx
const { icon: Icon } = props
// or 
const Icon = props.icon

// and then 

<Box>
  <Icon />
</Box> 
```